workflow-templates/docs.yml

```diff
@@ -0,0 +1,21 @@
+# Builds the Sphinx documentation in the docs folder
+
+---
+name: docs
+
+on: 
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - name: Sphinx install
+        run: pip install --upgrade sphinx
+      - name: Build doc
+        run: |
+		  cd docs
```

Comment:
Could this been used for other Sphinx dos than guides e.g. https://github.com/openmicroscopy/ome-documentation/ in which case it could be turned into a variable?

Semi-related, is a proposal that the Sphinx docs should  be under the `docs/` prefix by default moving forward?

Reply:
for the guides I have used the approach that GH is using for building the page i.e. ``docs/``

Reply:
:+1: apart from ome-model, bioformats and ome-documentation, do you know how many of our repositories do _not_ follow this layout?

Reply:
I have not noticed others, ``omero-figure``, ``omero-rois`` and even ``openmicroscopy`` use ``docs/``